ShadowrunAnarchy/translations/fr.json

```diff
@@ -90,5 +90,12 @@
     "whisper": "Murmurer",
     "wil": "VOL",
     "willpower": "Volonté",
-    "wounds": "Blessures"
+    "wounds": "Blessures",
+	"+S": "+F",
```

Comment:
These new keys also need to be added to the translation.json file in the top folder. Just adding them to the fr.json file will result in the other languages missing them and breaking for the sheet for those languages.

Reply:
I should have thought about it, indeed.
Done!